CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - Minor: Made join and part message have links to usercards. (#3358)
 - Minor: Show picked outcome in prediction badges. (#3357)
 - Minor: Add support for Emoji in IRC (#3354)
+- Minor: Moved `/live` logs to its own subdirectory. (Logs from before this change will still be available in `Channels -> live`). (#3393)
```

Comment:
```suggestion
- Minor: Moved `/live` logs to its own subdirectory. (Logs from before this change will still be available in `Channels/live`). (#3393)
```
Going to put this changelog variation here since it's probably more standard, but I do think `->` is very useful when explaining file paths to users.